json-endpoints/jetpack/class.jetpack-json-api-user-create-endpoint.php

```diff
@@ -26,16 +26,19 @@ function validate_input( $object ) {
 
 	function create_or_get_user() {
 		require_once JETPACK__PLUGIN_DIR . 'modules/sso/class.jetpack-sso-helpers.php';
-
 		// Check for an existing user
 		$user = get_user_by( 'email', $this->user_data['email'] );
-
+		/**
+		 * Fires after a user accepts an invitation and before a user gets created.
+		 * @return WP_User|false WP_User object on success, false on failure.
+		 */
+		do_action( 'jetpack_invitation_accepted', $user );
```

Comment:
It looks like this fires every time we create a user on this endpoint. We should be sure we don't use this API in other contexts.

Reply:
Good call! Will double check that.

Reply:
I think just to be on the safe side we should pass in a parameter from .com when this endpoint is used after a user accepts an invitation. 

Since it could be used in different to create users as well but is not at the moment.